Readme.md

```diff
@@ -213,7 +212,7 @@ secret, err := client.GetSecret(ctx, "store-name", "secret-name", opt)
 
 ## Service (callback)
 
-In addition to this Dapr API client, Dapr go SDK also provides `service` package to bootstrap your Dapr callback services in either gRPC or HTTP. Instructions on how to use it are located [here](./service/Readme.md)
+In addition to this Dapr API client operations, Dapr go SDK also provides `service` package to help you bootstrap Dapr callback services in either gRPC or HTTP. Instructions on how to use it are located [here](./service/Readme.md)
```

Comment:
rephrase `in addition to this Dapr API`

Reply:
rephrased

Reply:
`In addition to Dapr API client operations,...` remove this